tests/dummy/app/templates/docs/components/validated-button.md

```diff
@@ -5,7 +5,6 @@ accessed with `{{f.submit}}`. You can also use it as a block style component
 `{{#f.submit}}Test{{/f.submit}}` if you don't want to pass the label as a
 property. It takes the following properties:
 
-<div class="rounded border bg-grey-lightest my-8 p-4">
 **label `<String>`**
 <br>
```

Comment:
Wouldn't it be better/cleaner/more correct to use two trailing spaces on the preceding line instead of an HTML break?

```
**label `<String>`**••
The label of the form button.
```

Reply:
Good point, fixed!